examples/using-jest/__mocks__/gatsby.js

```diff
@@ -0,0 +1,14 @@
+const React = require('react')
+const gatsby = jest.requireActual('gatsby')
+
+module.exports = {
+  ...gatsby,
+  graphql: jest.fn(),
+  Link: jest.fn().mockImplementation(({ to, ...rest }) =>
```

Comment:
This is kinda janky, but I also think that this should _probably_ be added to the docs/unit-testing.md document!

Reply:
(also just for clarity here, gatsby-link errors out in Jest, so this is a way to side step that issue)